src/main/java/mate/academy/onlinebookstore01/model/Book.java

```diff
@@ -8,9 +8,13 @@
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Data
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted = false")

```

Comment:
The @where annotation is deprecated. 
You can use @SQLRestriction("is_deleted=false")

Reply:
Pay attention to this comment